shared-libs/couch-request/src/couch-request.js

```diff
@@ -18,18 +20,19 @@ const methods = {
   HEAD: 'HEAD'
 };
 
+const promisedAdminAuthHeaders = getAuthHeaders(environment.username, '_admin');
```

Comment:
This needs to be changed to not set the `_admin` role by default.  I have it like this so everything runs, but the intended approach is to just use the default username with its "member" access.  Then, for requests that need admin access, they will have to explicitly set the `_admin` role header.